spec/features/audit_spec.rb

```diff
@@ -0,0 +1,239 @@
+RSpec.feature "Audit management", type: :feature do
+  let!(:url_prefix) { "/#{@organization.to_param}" }
+
+  context "while signed in as a normal user" do
+    before do
+      sign_in(@user)
+    end
+
+    scenario "User can not visit the audits #index page" do
+      visit url_prefix + "/audits"
+      expect(page).to have_content("Access Denied")
+    end
+
+    scenario "User can not visit the audits #new page" do
+      visit url_prefix + "/audits/new"
+      expect(page).to have_content("Access Denied")
+    end
+
+    scenario "User can not visit the audits #edit page" do
+      visit url_prefix + "/audits/1/edit"
+      expect(page).to have_content("Access Denied")
+    end
+
+    scenario "User can not visit the audits #show page" do
+      visit url_prefix + "/audits/1"
+      expect(page).to have_content("Access Denied")
+    end
+  end
+
+  context "while signed in as an organization admin" do
+    before do
+      sign_in(@organization_admin)
+    end
+
+    scenario "User can filter the #index by storage location" do
+      storage_location = create(:storage_location, name: "here", organization: @organization)
+      storage_location2 = create(:storage_location, name: "there", organization: @organization)
+      create(:audit, organization: @organization, storage_location: storage_location)
+      create(:audit, organization: @organization, storage_location: storage_location2)
+
+      visit url_prefix + "/audits"
+      select storage_location.name, from: "filters_at_location"
+      click_button "Filter"
+
+      expect(page).to have_css("table tr", count: 2)
+    end
+
+    scenario "User can save progress of an audit" do
+      quantity = 7
+      storage_location = create(:storage_location, :with_items, organization: @organization)
+      visit url_prefix + "/audits"
+      click_link "New Audit"
+      select storage_location.name, from: "From storage location"
+      select Item.last.name, from: "audit_line_items_attributes_0_item_id"
+      fill_in "audit_line_items_attributes_0_quantity", with: quantity.to_s
+
+      expect do
+        click_button "Save Progress"
+        expect(Audit.last.in_progress?).to be_truthy
+        expect(Audit.last.line_items.count).to be(1)
+        expect(Audit.last.line_items.last.quantity).to be(7)
```

Comment:
Change the `7` here to `quantity`, since you defined it as that above (it avoids any "magic numbers")